src/hacbs/components/ApplicationDetails/tabs/CommitsTab.tsx

```diff
@@ -9,13 +9,17 @@ import {
   EmptyStateSecondaryActions,
 } from '@patternfly/react-core';
 import { OutlinedFileImageIcon } from '@patternfly/react-icons/dist/esm/icons/outlined-file-image-icon';
+import { getCommitsFromPLRs } from '../../../utils/commits-utils';
+import { pipelineWithCommits } from '../../Commits/__data__/pipeline-with-commits';
+import CommitsListView from '../../Commits/CommitsListView';
 
 type CommitTabProps = {
   applicationName: string;
 };
 
 const CommitsTab: React.FC<CommitTabProps> = ({ applicationName }) => {
-  return (
+  const commits = getCommitsFromPLRs(pipelineWithCommits);
```

Comment:
We need to remove the mock data and use actual pipelineruns
```suggestion
  const commits = getCommitsFromPLRs(pipelineWithCommits);
```